keyboards/ergo42/keymaps/hdbx/keymap.c

```diff
@@ -151,7 +197,13 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
   switch (keycode) {
     case QWERTY:
       if (record->event.pressed) {
-        persistent_default_layer_set(1UL<<_QWERTY);
+        set_single_persistent_default_layer(_QWERTY);
+      }
+      return false;
+      break;
+    case GAME:
+      if (record->event.pressed) {
+        set_single_persistent_default_layer(_GAME);
```

Comment:
Not sure that you want this as a persistent layer.  And from your keymap, you have other ways to get to it.